lib/elixir/lib/list.ex

```diff
@@ -133,6 +135,8 @@ defmodule List do
   Folds (reduces) the given list from the right with
   a function. Requires an accumulator.
 
+  Notice that `foldr/3` is not [tail-recursive](https://en.wikipedia.org/wiki/Tail_call). Therefore, `foldl/3` is usually preferred.
```

Comment:
Small nitpick: can you split this line into two lines? We do this in Elixir's documentation, as you can see in the two lines above these.